packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -734,3 +714,76 @@ async function areFoldersEmpty(folderPaths: string[]): Promise<boolean> {
 async function waitSeconds(second: number) {
   return new Promise((resolve) => setTimeout(resolve, second * 1000));
 }
+
+async function getDeploymentError(
+  deployCtx: DeployContext,
+  resourceGroupName: string,
+  deploymentName: string
+): Promise<any> {
+  let deployment;
+  try {
+    deployment = await deployCtx.client.deployments.get(resourceGroupName, deploymentName);
+  } catch (error) {
+    return undefined;
```

Comment:
return or throw error that mentions we failed to get deployment error?

Reply:
It is fixed. Thanks.
It will not be an error if the sub-deployment is not found. 
I will throw out error except this scenario.